llvm/utils/TableGen/TableGen.cpp

```diff
@@ -33,50 +33,47 @@ cl::opt<bool> EmitLongStrLiterals(
     cl::Hidden, cl::init(true));
 } // end namespace llvm
 
-namespace {
+static cl::OptionCategory PrintEnumsCat("Options for -print-enums");
```

Comment:
nit: I might be pedantic but I would prefer to split this anonymous namespace -> static refactoring to a separate commit / PR.

Reply:
Done, I started https://github.com/llvm/llvm-project/pull/105504. I'll update this diff once that gets merged.

Reply:
This is committed now.